app/views/tasks.html

```diff
@@ -102,30 +102,30 @@ <h4>{{'label.heading.pendingtasks' | translate}}</h4>
                             </div>
                             <script type="text/ng-template" id="deletechecker.html">
                                 <div class="modal-header silver">
-                                    <h3 class="bolder">{{'label.heading.delete' | translate}}</h3>
+                                    <h3 class="bolder">{{'label.heading.delete' | translate}}?</h3>
                                 </div>
                                 <div class="modal-body">
                                     <api-validate></api-validate>
                                     <br>
-                                    <button class="btn btn-default" ng-click="cancel()">{{'label.button.cancel' |
+                                    <button class="btn btn-default" ng-click="cancel()">{{'label.heading.no' |
                                         translate}}
                                     </button>
-                                    <button class="btn btn-primary" ng-click="delete()">{{'label.button.confirm' |
+                                    <button class="btn btn-primary" ng-click="delete()">{{'label.heading.yes' |
                                         translate}}
                                     </button>
                                 </div>
                             </script>
                             <script type="text/ng-template" id="approvechecker.html">
                                 <div class="modal-header silver">
-                                    <h3 class="bolder">{{'label.heading.approve' | translate}}</h3>
+                                    <h3 class="bolder">{{'label.heading.approve' | translate}}?</h3>
```

Comment:
same as above

Reply:
And also the issue says to change the pop up message like Confirm approve etc. Kindly check the issue for more details. And in reject popup also 'approve ' text is displayed. Kindly fix that.